sympy/core/exprtools.py

```diff
@@ -1299,7 +1299,9 @@ def factor_nc(expr):
     """Return the factored form of ``expr`` while handling non-commutative
     expressions.
 
-    **examples**
```

Comment:
I meant to say "oh while you're at fixing stuff manually, === is more common than --- in this case".
I fully agree that making that kind of correction automatic is probably overeager.